linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -610,6 +610,10 @@ private[optimizer] abstract class OptimizerCore(
           pretransformExpr(tree)(finishTransform(isStat))
         }
 
+      case prop: LinkTimeProperty =>
+        config.coreSpec.linkTimeProperties
+          .transformLinkTimeProperty(prop)
```

Comment:
This comfortably fits on one line:
```suggestion
        config.coreSpec.linkTimeProperties.transformLinkTimeProperty(prop)
```